code/datums/components/archaeology.dm

```diff
@@ -1,13 +1,19 @@
 /datum/component/archaeology
 	dupe_mode = COMPONENT_DUPE_UNIQUE
-	var/list/archdrops
+	var/list/archdrops = list(/obj/item/bikehorn = list(ARCH_PROB = 100, ARCH_MAXDROP = 1)) // honk~
 	var/prob2drop
 	var/dug
 	var/datum/callback/callback
 
-/datum/component/archaeology/Initialize(_prob2drop, list/_archdrops = list(), datum/callback/_callback)
-	prob2drop = CLAMP(_prob2drop, 0, 100)
+/datum/component/archaeology/Initialize(list/_archdrops = list(), datum/callback/_callback)
 	archdrops = _archdrops
+	for(var/i in archdrops)
+		if(isnull(archdrops[i][ARCH_MAXDROP]))
+			archdrops[i][ARCH_MAXDROP] = 1
+			stack_trace("ARCHAEOLOGY WARNING: [parent] contained a null max_drop value in [i].")
+		if(isnull(archdrops[i][ARCH_PROB]))
+			archdrops[i][ARCH_PROB] = 100
+			stack_trace("ARCHAEOLOGY WARNING: [parent] contained a null probability value in [i].")
```

Comment:
is this ok or is this "too" conservative?

Reply:
define what you mean by too conservative

Reply:
I was thinking this wouldn't ever happen outside of badminnery since the lists are relatively static in the current iteration, but now that I think about it if it becomes more dynamic (IE we later want Boss dying = increased chances of certain items/new loot) then this is probably good to have.